src/language.ts

```diff
@@ -70,9 +86,11 @@ export const getChangeLanguageWebChat = (manager: FlexWebChat.Manager, config: C
         ...languageTranslations,
         ...configTranslations[language],
       });
+      setMainHeaderTitle(manager, defaultLanguage);
     } else {
       setConfigLanguage(defaultLanguage);
       setNewStrings({ ...twilioStrings, ...defaultLanguageTranslations, ...configTranslations[defaultLanguage] });
+      setMainHeaderTitle(manager, defaultLanguage);
```

Comment:
-  setMainHeaderTitle(manager, language);  <=== it should be `language` in the `if` case
-  setMainHeaderTitle(manager, defaultLanguage);   <=== it should be `defaultLanguage` in the `else` case

Reply:
@murilovmachado, I see what you mean. Sorry for the back and forth. It's done.